src/patternfly/components/Badge/badge.hbs

```diff
@@ -2,5 +2,10 @@
   {{#if badge--attribute}}
     {{{badge--attribute}}}
   {{/if}}>
-  {{> @partial-block}}
+  {{#if badge--text}}
+    {{badge--text}}
+  {{/if}}
+  {{#if @partial-block}}
```

Comment:
just a nit, but if the plan is to use `--text` or `@partial-block`, and not together, having `@partial-block` as an `else` condition will keep this partial from spitting stuff out if it's included as a child of a parent HBS partial that takes `@partial-block` that is intended for the parent. Written the way it is in the PR we'd need to do `{{#> badge badge--text="foo"}}{{/badge}}`. With an `else` condition here we could just do `{{> badge badge-text="foo"}}`